app/authentication/user.py

```diff
@@ -1,8 +1,10 @@
+from typing import Any, Optional
+
 from flask_login import UserMixin
 
 
 class User(UserMixin):
-    def __init__(self, user_id, user_ik):
+    def __init__(self, user_id: str, user_ik: Optional[Any]) -> None:
```

Comment:
Is user_ik not a str?

Reply:
Yeah I initially had it set as `str` but it generates the following mypy error `"User" has incompatible type "Optional[Any]"; expected "str"`.

Reply:
Mine works with Optional[str], maybe need someone else to try

Reply:
It works for me so have updated it